src/main/java/org/graylog/integrations/notifications/types/SlackEventNotification.java

```diff
@@ -154,27 +159,38 @@ private String buildMessageTitle(EventNotificationContext ctx) {
 
 	String buildCustomMessage(EventNotificationContext ctx, SlackEventNotificationConfig config, String template) throws PermanentEventNotificationException {
 		List<MessageSummary> backlog = getAlarmBacklog(ctx);
-		Map<String, Object> model = getCustomMessageModel(ctx, config, backlog);
-		try {
-			LOG.debug("template = {} model = {}" ,template, model);
-			return templateEngine.transform(template, model);
-		} catch (Exception e) {
-			LOG.error("Exception during templating [{}]", e.toString());
-			throw new PermanentEventNotificationException(e.toString(),e.getCause());
+		backlog = getMessageBacklog(config, backlog);
+
+		Map<String, Object> model = getCustomMessageModel(ctx, config.type(), backlog);
+			try {
+				LOG.debug("template = {} model = {}", template, model);
+				return templateEngine.transform(template, model);
+			} catch (Exception e) {
+				LOG.error("Exception during templating [{}]", e.toString());
+				throw new PermanentEventNotificationException(e.toString(), e.getCause());
+		}
+
+
+	}
+
+	List<MessageSummary> getMessageBacklog(SlackEventNotificationConfig config, List<MessageSummary> backlog) {
```

Comment:
We usually annotate package-private methods with a `@VisibleForTesting` annotation to make other developers know why the method isn't private.